src/utils/server-utils.ts

```diff
@@ -111,9 +116,8 @@ async function getWMSCapabilities(serverUri: string) {
 
     return formatCapabilitiesInfo(flatLayers, 'Name._text', 'Dimension._text');
   } catch (error) {
-    throw new Error(
-      `Failed loading dates with request GET: ${requestUri} for WMS Capabilities, ${error}`,
-    );
+    console.error(error);
```

Comment:
why this change?

Reply:
#87 introduced this error in order to show a notification if the request to the WMS or WCS server failed. However, if we throw an error if one fails even if the others succeed, then dates aren't loaded for anything else such as groundstation layers. Currently the WCS API url is failing: https://github.com/WFP-VAM/prism-frontend/blob/50752fc36395c2b60a51139e3ef04306200aa7c5/src/config/prism.json#L4, meaning nothing else is allowed to load. Removing the error and just printing bypasses this.

This is moreso a temporary solution, what should we do?

Reply:
That's fine for now then but maybe add a todo?